### ISSUE TITLE
Two github accounts linked same repository, breaking some relation in database

### ISSUE THREAD

**Issue by kklimonda-fn** [cla-assistant/cla-assistant#932]
The repository `fluxninja/aperture` has been linked under two accounts: kklimonda-fn and fluxninjaops. Doing that resulted in repository showing under both accounts in cla-assistant.io but only first one (kklimonda-fn) sees link to CLA gist and a list of signees.

**Comment by KharitonOff** [comment]
@kklimonda-fn this is correct behavior if fluxninjaops has 'write' rights on the repository, but doesn't belong to the administrators.
